src/test/java/org/springframework/retry/annotation/RecoverAnnotationRecoveryHandlerTests.java

```diff
@@ -16,6 +16,7 @@
 
 package org.springframework.retry.annotation;
 
+import java.lang.annotation.*;
```

Comment:
Please, don't use an asterisk imports.
Please, add your name to the `@author` list of all the effected classes.
And update their Copyright to the current year.